### PR TITLE
Support multiple inference providers (multiple LLMs)

### PR DESCRIPTION
**Pull request by vivekuppal**
Add support for 
- OpenAI
- Together

### CODE REVIEW

File: app/transcribe/app_utils.py [hunk 1]
```diff
@@ -3,13 +3,35 @@
...
+                                                             save_to_file=save_to_file,
+                                                             file_name=file_name)
+    elif provider_name.lower() == 'together':
+        responder = responder_factory.get_stt_model_instance(provider=InferenceEnum.TOGETHER,
+                                                             config=config,
+                                                             convo=convo,
+                                                             save_to_file=save_to_file,
+                                                             file_name=file_name)
+    else:
+        responder = None
```
Comment (IAmADolphin) [inline app/transcribe/app_utils.py:31]: Handle None case from calling method

File: app/transcribe/args.py [hunk 2]
```diff
@@ -23,6 +23,9 @@ def create_args() -> argparse.Namespace:
                           help='Specify the Speech to text Engine.'
                           '\nLocal STT models tend to perform best for response times.'
                           '\nAPI based STT models tend to perform best for accuracy.')
+    cmd_args.add_argument('-c', '--chat', action='store', default='openai',
```
Comment (IAmADolphin) [inline app/transcribe/args.py:26]: --chat-inference is a better name

**Review by IAmADolphin — APPROVED** [review]
Good besides the comments

File: app/transcribe/app_utils.py (hunk 1 above)
Comment (vivekuppal) [reply app/transcribe/app_utils.py:31]: Done, Thx!

File: app/transcribe/args.py (hunk 2 above)
Comment (vivekuppal) [reply app/transcribe/args.py]: Done

**Pull request merged**